src/shared-components/dropdown/desktopDropdown.tsx

```diff
@@ -84,10 +86,10 @@ const DesktopDropdown = ({
         <DropdownOptionsContainer
           isOpen={isOpen}
           optionsContainerMaxHeight={optionsContainerMaxHeight}
-          role="listbox"
+          role="menu"
           aria-activedescendant={value}
-          tabIndex={isOpen ? 0 : -1}
```

Comment:
Removed the `tabindex` from the `ul` - this felt like a weird intermediate step to focus on the entire menu list instead of simply proceeding to the element in the list.